generator_layout.html

```diff
@@ -0,0 +1,907 @@
+
+                    <button type="button" class="add-line">
+                        {{ page.technical_info.evidence.add_button_text }}
+                    </button>
+                </div>
+
+                {% include excol.html type="end" %}
+
+                {% include excol.html type="start" %}
+                <h2 id="create-approval">{{ page.approval_and_complaints.title }}</h2>
+                <p class="intro">{{ page.approval_and_complaints.info }}</p>
+                {% include excol.html type="middle" %}
+
+                <h3 class="label">{{ page.approval_and_complaints.approval.title }}
+                    <button type="button" class="showhidebutton button-small" aria-expanded="false"
+                        aria-controls="org_approved" data-target="#org_approved" data-showtext="{{ page.show_info }}"
+                        data-hidetext="{{ page.hide_info }}">{{ page.show_info }}</button>
+                </h3>
+                <div class="information" id="org_approved" hidden="hidden">
+                    <p>{{ page.approval_and_complaints.approval.info }}</p>
+                </div>
+
+                <fieldset class="group">
+                    <legend class="visuallyhidden">{{ page.approval_and_complaints.approval.title }}</legend>
+
+                    <div class="field">
+                        <label for="accstmnt_orginfo_approved_name">{{ page.approval_and_complaints.approval.name.title }}</label>
+                        <em>{{ page.approval_and_complaints.approval.name.example }}</em>
+                        <input type="text" id="accstmnt_orginfo_approved_name" />
+                    </div>
+
+                    <div class="field">
+                        <label for="accstmnt_orginfo_approved_function">{{ page.approval_and_complaints.approval.function.title }}</label>
+                        <em>{{ page.approval_and_complaints.approval.function.example }}</em>
+                        <input type="text" id="accstmnt_orginfo_approved_function" />
+                    </div>
+                </fieldset>
+
+                <h3 class="label">{{ page.approval_and_complaints.complaints.title }}
+                    <button type="button" class="showhidebutton button-small" aria-expanded="false"
+                        aria-controls="info_complaints" data-target="#info_complaints" data-showtext="{{ page.show_info }}"
+                        data-hidetext="{{ page.hide_info }}">{{ page.show_info }}</button>
+                </h3>
+                <div class="information" id="info_complaints" hidden="hidden">
+                    <p>{{ page.approval_and_complaints.complaints.info }}</p>
+                </div>
+
+                <div class="field">
+                    <label for="accstmt_complaints">{{ page.approval_and_complaints.complaints.label }}</label>
+                    <textarea id="accstmt_complaints" rows="3"></textarea>
+                </div>
+                {% include excol.html type="end" %}
```

Comment:
Please switch order of the blocks for "approval" and "complaints" -- "complaints" should appear before "approval"
(addresses issue #109)